exporter/otlpexporter/README.md

```diff
@@ -28,6 +28,16 @@ The following settings can be optionally configured:
   [grpc.WithInsecure()](https://godoc.org/google.golang.org/grpc#WithInsecure).
 - `balancer_name`(default = pick_first): Sets the balancer in grpclb_policy to discover the servers.
 See [grpc loadbalancing example](https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md).
+- `timeout` (default = 5s): Is the timeout for each operation.
```

Comment:
It would be good to have further explanation on the relationship between the timeout and the retry/queued settings. For instance, is this timeout valid for the whole processor, or is it per-retry? "Each operation" is ambiguous, as it can mean "every trace/metrics data that was received" and "every attempt to send trace/metrics".

Reply:
Good point. It actually is "every attempt to send data to the backend"